demos/storybook/.gitignore

```diff
@@ -0,0 +1,65 @@
+# Logs
```

Comment:
storybook should not have it's own gitignore...everything should be handled by the one at the root